pkg/localkube/ready.go

```diff
@@ -50,8 +50,8 @@ func healthCheck(addr string, lk LocalkubeServer) HealthCheck {
 			RootCAs:      caCertPool,
 		}
 		tlsConfig.BuildNameToCertificate()
-		transport := &http.Transport{TLSClientConfig: tlsConfig}
-		client := &http.Client{Transport: transport}
+		transport := lk.GetTransport()
```

Comment:
This function doesn't exist yet - you'll have to rip out the relevant parts from the existing `healthCheck` function